src/Apps/Order2/Routes/Details/Components/Order2DetailsMessage.tsx

```diff
@@ -10,23 +11,301 @@ export const Order2DetailsMessage = ({ order }: Order2DetailsMessageProps) => {
   const orderData = useFragment(FRAGMENT, order)
 
   return (
-    <Stack gap={1}>
-      {orderData.displayTexts.message && (
-        <Text
-          variant="sm"
-          dangerouslySetInnerHTML={{
-            __html: orderData.displayTexts.message,
-          }}
-        />
-      )}
-    </Stack>
+    <Box backgroundColor="mono0" p={2}>
+      {getMessageContent(orderData)}
+    </Box>
   )
 }
 
+const YourCollectionNote: React.FC = () => (
+  <Text variant="sm">
+    This artwork will be added to{" "}
+    <RouterLink to="/collector-profile/my-collection" display="inline">
+      your Collection on Artsy
+    </RouterLink>
+    .
+  </Text>
+)
+
+const NumberedListItem: React.FC<{
```

Comment:
Maybe worth creating a palette component for `numberedList` eventually but for now putting it here similar to how the App does it.